packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -222,25 +224,23 @@ const retailReactAppPrompts = () => {
 
     const questions = [
         {
-            name: 'projectId',
-            validate: validProjectId,
-            message: 'What is your project ID (example-project) in Managed Runtime Admin?'
+            name: 'projectName',
+            validate: validProjectName,
+            message: 'What is the name of your Project?'
         },
         {
             name: 'instanceUrl',
-            message:
-                'What is the URL (https://example_instance_id.sandbox.us01.dx.commercecloud.salesforce.com) for your B2C Commerce instance?',
+            message: 'What is the URL for your Commerce Cloud instance?',
             validate: validUrl
         },
         {
             name: 'clientId',
-            message: 'What is your API client ID?',
+            message: 'What is your Commerce API client ID in Account Manager?',
             validate: validClientId
         },
         {
             name: 'siteId',
-            message:
-                "What is your site's ID (examples: RefArch, RefArchGlobal) in Business Manager?",
+            message: "What is your site's ID in Business Manager?",
```

Comment:
This is typically referred to as "Site ID" in BM:

<img width="1695" alt="Screen Shot 2022-02-02 at 9 38 33 AM" src="https://user-images.githubusercontent.com/28967/152207321-def4b141-6f46-4ee3-83cc-8ab0f081885f.png">

Reply:
Same, thanks for digging in what the users see. I'll confirm the copy with @jennmills.